.bcr/metadata.template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "homepage": "https://github.com/aranguyen/rules_platform",
+  "maintainers": [
+    {
+      "name": "Ara Nguyen",
+      "email": "aranguyen@google.com",
+      "github": "aranguyen"
```

Comment:
What are other bazel team projects doing? Should this be Open Source Team? blaze-configurability-team@google.com?

Reply:
This is what `rules_java` has https://github.com/bazelbuild/rules_java/blob/master/.bcr/metadata.template.json. I'll do the same